src/app/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Miscellaneous
```

Comment:
Shouldn't the gitignore be at the root of the git repository?

Reply:
That's what I was wondering. In Flutter we have a `.gitignore` in each project folder and one at the root of the project, may be I should just move the one in the app folder into the root?

Reply:
@CBrunet2212

Reply:
I deleted the one at the project root. Kept the other as they are because moving them broke the whole thing. If we wish to have them at the root for whatever reason in the future we should dedicate a work item for this alone.

Reply:
No you should probably not remove the gitignore that is at the root of the repo.